docker/polygon_zkevm/polygon_zkevm.sh

```diff
@@ -43,6 +45,21 @@ create_deployment_config_file () {
     cd ${ROOT}
 }
 
+fund_account() {
+  echo "Fund account"
+  DEPOSIT_FILE="${POLYGON_ZKEVM}/test/scripts/deposit/main.go"
+  # Replace the hardcoded address and private key with our L1 account
+  sed -i'' -e 's/0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266/0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC/' "${DEPOSIT_FILE}"
+  sed -i'' -e 's/0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80/0x5de4111afa1a4b94908f83103eb1f1706367c2e68ca870fc3fb9a804cdab365a/' "${DEPOSIT_FILE}"
+
+  cd ${POLYGON_ZKEVM}
+  go run test/scripts/deposit/main.go
```

Comment:
how this didnt fail at the first place? i am pretty sure it will fail as soon as we changed the cache key.

Reply:
the vm we use has go in it. https://discuss.circleci.com/t/linux-machine-executor-update-2023-q1-update/47059